ibc/host.go

```diff
@@ -0,0 +1,25 @@
+package ibc
+
+import (
+	"time"
+
+	"github.com/pokt-network/pocket/shared/modules"
+)
+
+var _ modules.IBCHost = (*Host)(nil)
```

Comment:
I know we discussed this offline, but I do think we should do `&Host{}` for consistency in the codebase.